index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
```

Comment:
We already have this in the dependencies. Please move these to the related component. (MapView.js)
You don't have to use a CDN, you could just do, inside MapView, `import 'leaflet/dist/leaflet.css'`